src/observers/ReportIssueObserver.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { vscode } from "../vscodeAdapter";
+import { BaseEvent, ReportIssue } from "../omnisharp/loggingEvents";
+
+export class ReportIssueObserver {
+
+    constructor(private vscode: vscode) {
+    }
+
+    public post = (event: BaseEvent) => {
+        switch (event.constructor.name) {
+            case ReportIssue.name:
+                let issue = <ReportIssue>event;
```

Comment:
I'm not sure how I feel about needing to dispatch to the message bus to do this...

Reply:
What I mean is that there could be a component that's basically identical to the observer youa dded, but they could be directly composed together as opposed to going over the message bus.